### PR TITLE
Feature/enable unstructured

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
## PR Overview
**This PR will address the following Issue/Feature:**
- Internal

**This PR will result in the following new package version:**

- v0.19.0 due to schema change

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

> ## Breaking Changes
> - Model `zendesk__document` is now enabled by default. To disable it, by set the `zendesk__unstructured_enabled` variable to `false` in your `dbt_project.yml`.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] ~dbt run (if incremental models are present) && dbt test~ no changes to incremental

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

- testing without setting any variables confirms `zendesk__document` is created.
Screenshot 2024-10-10 at 1 40 31 PM

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
-# dbt_zendesk v0.17.0
+# dbt_zendesk v0.19.0
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:1]: Let's make this a pre-release
```suggestion
# dbt_zendesk v0.19.0-a1
```

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:1]: Can you also make the same update in the relevant sections of the documentation.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+[PR #171](https://github.com/fivetran/dbt_zendesk/pull/172) includes the following changes:
+## Breaking Changes
+- Model `zendesk__document` is now enabled by default. To disable it, by set the `zendesk__unstructured_enabled` variable to `false` in your `dbt_project.yml`.
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:4]:
```suggestion
- Model `zendesk__document` is now enabled by default. To disable it, set the `zendesk__unstructured_enabled` variable to `false` in your `dbt_project.yml`.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
-# dbt_zendesk v0.17.0
+# dbt_zendesk v0.19.0
+[PR #171](https://github.com/fivetran/dbt_zendesk/pull/172) includes the following changes:
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:2]:
```suggestion
[PR #173](https://github.com/fivetran/dbt_zendesk/pull/173) includes the following changes:
```

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
A few change requests before this will be good. Also, with this being a pre-release we may want to hold off merging anything since we'll likely cut the release off the branch.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM! Last request here would be to update the README to explicitly callout the pre-release version (like we did [here](https://github.com/fivetran/dbt_zendesk/tree/v0.8.0-b1?tab=readme-ov-file#installation-instructions) for `v0.8.0-b1`).

Once that's updated this should be good to create a pre-release off the release branch!

**Review by fivetran-jamie — APPROVED** [review]
lgtm!

**Pull request merged**